components/ImprovedSearch/Button.tsx

```diff
@@ -0,0 +1,20 @@
+import { forwardRef } from 'react'
+import styled from 'styled-components'
+import { Button as EdsButton, ButtonProps as EdsButtonProps } from '@equinor/eds-core-react'
+
+export type ButtonProps = EdsButtonProps
+
+const StyledEdsButton = styled(EdsButton)`
+  --eds_button__font_size: var(--typeScale-05);
```

Comment:
We don't have these CSS variables in our code base, so this doesn't actually do anything 🤷‍♀️  I think you should reconsider if we need this button as a separate component at all